common/app/common/Edition.scala

```diff
@@ -13,6 +13,7 @@ abstract class Edition(
     val timezone: DateTimeZone,
     val locale: Locale,
     val homePagePath: String,
```

Comment:
It appears to be used in a few other places so haven't removed it however the `homePagePath` is now derived from `networkFrontId` by first prepending a '/'. The `toLowerCase` method has also been removed as suggested.